contrib/cf-remote/README.md

```diff
@@ -1,33 +1,84 @@
+```
+
+Check that it worked:
+
 ```
-$ cf-remote info -H 192.168.100.90
+$ ls -al `which cf-remote`
+lrwxr-xr-x  1 olehermanse  admin  68 Jan 10 15:20 /usr/local/bin/cf-remote -> /northern.tech/cfengine/core/contrib/cf-remote/cf_remote/__main__.py
+$ cf-remote --version
+cf-remote version 0.1 (BETA)
+Available CFEngine versions:
+3.13.0, 3.12.1, 3.12.0, 3.10.5, 3.10.4, 3.10.3, 3.10.2, 3.10.1, 3.10.0
 ```
 
-## Ideas
+Note that the version/branch of core you have checked out is important!
+You can also move it out of your core folder if you prefer this (but then making changes to cf-remote becomes more tedious).
+
+## Examples
+
+### See information about remote host
 
-Useful example:
+The `info` command can be used to check basic information about a system.
 
 ```
-$ cf-remote install --hub 192.168.100.90 --client 192.168.100.91 --bootstrap 192.168.100.90
+$ cf-remote info -H 34.241.203.218
+
+ubuntu@34.241.203.218
+OS            : ubuntu (debian)
+Architecture  : x86_64
+CFEngine      : 3.12.1
+Policy server : 172.31.42.192
+Binaries      : dpkg, apt
 ```
 
-`--bootstrap` has an optional argument in case you want to bootstrap to a different IP than `hub`.
-For example, in AWS you should bootstrap to the internal IP.
+(You must have ssh access).
+
+### Installing and bootstrapping CFEngine Enterprise Hub
+
+The `install` command can automatically download and install packages as well as bootstrap both hubs and clients.
 
-Ideas for commands:
 ```
-$ cf-remote install
-$ cf-remote status
-$ cf-remote info
-$ cf-remote vars
-$ cf-remote classes
-$ cf-remote execute
-$ cf-remote restart
-$ cf-remote uninstall
-$ cf-remote help
-$ cf-remote version
-$ cf-remote --version
-$ cf-remote --help
+cf-remote install --hub 34.247.181.100 --bootstrap 172.31.44.146 --demo
+
+ubuntu@34.247.181.100
+OS            : ubuntu (debian)
+Architecture  : x86_64
+CFEngine      : Not installed
+Policy server : None
+Binaries      : dpkg, apt
+
+Package already downloaded: '/Users/olehermanse/.cfengine/cf-remote/packages/cfengine-nova-hub_3.12.1-1_amd64.deb'
+Copying: '/Users/olehermanse/.cfengine/cf-remote/packages/cfengine-nova-hub_3.12.1-1_amd64.deb' to '34.247.181.100'
+Installing: 'cfengine-nova-hub_3.12.1-1_amd64.deb' on '34.247.181.100'
+CFEngine 3.12.1 was successfully installed on '34.247.181.100'
+Bootstrapping: '34.247.181.100' -> '172.31.44.146'
+Bootstrap succesful: '34.247.181.100' -> '172.31.44.146'
+Transferring def.json to hub: '34.247.181.100'
+Copying: '/Users/olehermanse/.cfengine/cf-remote/json/def.json' to '34.247.181.100'
+Triggering an agent run on: '34.247.181.100'
+Disabling password change on hub: '34.247.181.100'
+Triggering an agent run on: '34.247.181.100'
+Your demo hub is ready: https://34.247.181.100/ (Username: admin, Password: password)
 ```
+
+Note that this demo setup is notoriously insecure.
+Please change passwords and access controls in both Mission Portal and policy before using CFEngine in a production environment.
```

Comment:
I'd suggest not using it at all for a production environment.

Reply:
Thanks. Changed the wording.